src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package com.endava.internship.mocking.service;
+
+
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Why do you need @ExtendWith(MockitoExtension.class) here?

Reply:
I forgot to delete it